src/tests/uat/reputation_override_uat.py

```diff
@@ -0,0 +1,160 @@
+# *******************************************************
+# Copyright (c) VMware, Inc. 2021. All Rights Reserved.
+# SPDX-License-Identifier: MIT
+# *******************************************************
+# *
+# * DISCLAIMER. THIS PROGRAM IS PROVIDED TO YOU "AS IS" WITHOUT
+# * WARRANTIES OR CONDITIONS OF ANY KIND, WHETHER ORAL OR WRITTEN,
+# * EXPRESS OR IMPLIED. THE AUTHOR SPECIFICALLY DISCLAIMS ANY IMPLIED
+# * WARRANTIES OR CONDITIONS OF MERCHANTABILITY, SATISFACTORY QUALITY,
+# * NON-INFRINGEMENT AND FITNESS FOR A PARTICULAR PURPOSE.
+
+"""
+The following API calls are tested in this script.
+
+For the validation CBC SDK Select are used.
+
+To execute, a profile must be provided using the standard CBC Credentials.
+
+Reputation Override:
+* Configure Reputation Override
+* Get Reputation Override
+* Delete Reputation Override
+* Search Reputation Overrides
+* Bulk Delete Reputation Overrides
+
+"""
+
+# Standard library imports
+import sys
+
+# Internal library imports
+from cbc_sdk.helpers import build_cli_parser, get_cb_cloud_object
+from cbc_sdk.platform import ReputationOverride
+from cbc_sdk.errors import ObjectNotFoundError
+
+# Formatters
+NEWLINES = 1
+DELIMITER = '-'
+SYMBOLS = 48
```

Comment:
Those are not used, same for the ORG_KEY and HOSTNAME in the main function.

Reply:
Thanks and fixed!